pandas/core/resample.py

```diff
@@ -937,25 +936,27 @@ def asfreq(self, fill_value=None):
         """
         return self._upsample("asfreq", fill_value=fill_value)
 
-    def std(self, ddof=1, *args, **kwargs):
+    def std(self, ddof=1, numeric_only: bool = False):
```

Comment:
@mroeschke - I believe you've done a bunch of work in resample/window; do you know why args/kwargs exist on these ops? They seem to be on all these methods only to check that the user didn't pass anything to them. I'm going to revert these removals here, but was thinking about doing a cleanup of these in a separate PR if they aren't useful.

Reply:
I think the numpy compatibility was such that one could call e.g. `np.mean(df.groupby/resample/rolling)` and get the equivalent call to `df.groupby/resample/rolling.mean()`. Not sure why that compatibility was desired or if it was widely tested (guessing not if tests are passing).

FWIW I think I broke this compatibility too when I added engine keywords to the window and groupby reduction ops, so I'd be +1 to clean these *args, **kwargs